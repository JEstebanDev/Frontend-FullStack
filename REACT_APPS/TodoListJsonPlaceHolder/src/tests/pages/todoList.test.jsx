import { render } from "@testing-library/react";
import TodoList from "../../pages/todoList";

describe("testing the component", () => {
  test("should render the TodoList and take snapshot", () => {
    const { container } = render(<TodoList></TodoList>);
    expect(container).toMatchSnapshot();
  });
});
