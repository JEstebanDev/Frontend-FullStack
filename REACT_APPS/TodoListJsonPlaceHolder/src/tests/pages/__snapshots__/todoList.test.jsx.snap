// Vitest Snapshot v1

exports[`testing the component > should render the TodoList and take snapshot 1`] = `
<div>
  <div
    class="listCards animate__animated animate__fadeIn"
  />
</div>
`;
